.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "external/aos_core_common_cpp"]
+	path = external/aos_core_common_cpp
+	url = https://github.com/MykolaSolyanko/aos_core_common_cpp.git
+[submodule "external/api/aos_core_api"]
```

Comment:
Shall aos core lib be added at same level here?

Reply:
aos core lib is a submodule of common lib. So common lib includes core lib as a submodule